saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -855,7 +870,28 @@ public Builder assertionConsumerServiceBinding(Saml2MessageBinding assertionCons
 		 * @since 5.6
 		 */
 		public Builder singleLogoutServiceBinding(Saml2MessageBinding singleLogoutServiceBinding) {
-			this.singleLogoutServiceBinding = singleLogoutServiceBinding;
+			return this.singleLogoutServiceBindings((saml2MessageBindings) -> {
```

Comment:
Is there a reason the new method needs to be called? I'm wondering if

```java
this.singleLogoutServiceBindings = Collections.singletonList(singleLogoutServiceBinding);
```

would work?